docs/release_notes/october_2023.md

```diff
@@ -0,0 +1,34 @@
+# October 2023
+## airbyte v0.50.31 to v0.50.33
+
+This page includes new features and improvements to the Airbyte Cloud and Airbyte Open Source platforms.
+
+## **✨ Highlights
+
+This month, we held our annual Hacktoberfest, in which we received 48 PRs and welcomed 10 new contributors to our community!
+
+We also released [versioned Connector Documentation](https://github.com/airbytehq/airbyte/pull/30410), which allows everyone to see the correct version of the documentation for their connector version without needing to upgrade their Airbyte platform version.
+
+We're also always learning and listening to user feedback. We no longer [deduplicate raw tables](https://github.com/airbytehq/airbyte/pull/31520) to further speed up syncs with Destinations V2. We also released a new voting feature on our [docs](docs.airbyte.com) that asks how helpful our docs are for you.
```

Comment:
Needs to be `[docs](https://docs.airbyte.com)` without schema this will try to find a relative page docs.airbyte.com, which is what currently fails the build.